docs/pages/eas/workflows/syntax.mdx

```diff
@@ -210,6 +210,52 @@ jobs:
         run: pwd # prints: /home/expo/workingdir/build/my-app
 ```
 
+## `defaults`
+
+Parameters to use as defaults for all jobs defined in the workflow configuration.
+
+### `defaults.run.working_directory`
+
+Default working directory to run the scripts in. Relative paths like "./assets" or "assets" are resolved from the app's base directory. Absolute paths like "/apps/mobile" are resolved from the repository root.
```

Comment:
```suggestion
Default working directory to run the scripts in. Relative paths like "./assets" or "assets" are resolved from the app's base directory.
```

Ugh, I think this "absolute paths resolved from repository root" thing is quite clever, but I worry it's a bit too clever. Is it odd I don't want to put this into official docs? (I know it's in code syntax.)